templates/cpp-template-default/proj.android-studio/app/proguard-rules.pro

```diff
@@ -15,3 +15,22 @@
 #-keepclassmembers class fqcn.of.javascript.interface.for.webview {
 #   public *;
 #}
+
+# Proguard Cocos2d-x for release
+-keep public class org.cocos2dx.** { *; }
+-dontwarn org.cocos2dx.**
+-keep public class com.chukong.** { *; }
+-dontwarn com.chukong.**
```

Comment:
Please don't obfuscate the package name in `EngineDataManager.jar`.
They are:
com.huawei.android.hwgamesdk
org.cocos2dx.enginedata (this one isn't needed since it's included in org.cocos2dx.**)

Reply:
OK. Do you want that for just `cpp-template-default` or all?

Reply:
Sure, it's needed for all projects including templates.